tensorflow_lite_support/python/task/processor/proto/class_pb2.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Class protobuf."""
+
+from tensorflow_lite_support.cc.task.vision.proto import class_pb2
```

Comment:
Can this be put under a python/task/vision/proto/ folder instead, to match the C++ namespacing? I'm aware this is not consistent with what was done for ImageEmbedder, but imho this is better practice (and we should update the image embedder to match, cc @ziyeqinghan)

Reply:
Fixed it!